projects/1.18/assets/materialis/materialis/lang/zh_cn.json

```
@@ -1,195 +1,231 @@
+  "material.materialis.refined_glowstone.encyclopedia": "加上采掘速度，乘以效果等级，除以80，等于攻击速度",
   "material.materialis.refined_obsidian": "强化黑曜石",
   "material.materialis.refined_obsidian.encyclopedia": "-20%触及距离",
-  "material.materialis.refined_obsidian.format": "强化黑曜石%s",
   "material.materialis.refined_radiance": "光辉石",
   "material.materialis.refined_radiance.encyclopedia": "对目标造成20秒的发光效果，并在采掘方块后留下光源",
   "material.materialis.refined_radiance.format": "光辉石%s",
+  "material.materialis.regalium": "贵豪",
   "material.materialis.rose_quartz": "玫瑰石英",
   "material.materialis.rose_quartz.encyclopedia": "每个部件提供1个额外的升级槽",
   "material.materialis.rose_quartz.flavor": "免责声明: 实际上并非使用玫瑰制成。",
-  "material.materialis.rose_quartz.format": "玫瑰石英%s",
   "material.materialis.shadow_steel": "暗影钢",
   "material.materialis.shadow_steel.encyclopedia": "击杀的生物和采掘的方块都不再掉落物品，但是击杀生物后会掉落额外的经验值",
-  "material.materialis.shadow_steel.format": "暗影钢%s",
   "material.materialis.starmetal": "星辉",
   "material.materialis.starmetal.encyclopedia": "工具会在午夜获得每级2点的采掘速度",
   "material.materialis.starmetal.flavor": "不要与星形切割金属混淆。",
-  "material.materialis.starmetal.format": "星辉%s",
-  "material.materialis.tungsten": "钨",
-  "material.materialis.tungsten.encyclopedia": "工具的耐久度越低，耐久度的消耗几率越低，最低为1%",
-  "material.materialis.tungsten.format": "钨%s",
+  "material.materialis.terrasteel": "泰拉钢",
+  "material.materialis.terrasteel.encyclopedia": "每级有40%的概率发射一束激光，并消耗魔力（无魔力时则消耗耐久）",
   "material.materialis.uranium": "铀",
-  "material.materialis.uranium.encyclopedia": "耐久度在一半以上的时候会以双倍速度消耗耐久度，在低于一半耐久度的时候耐久度消耗速度则会越来越慢",
-  "material.materialis.uranium.format": "铀%s",
+  "material.materialis.uranium.encyclopedia": "工具在高于四分之一耐久时受到的耐久损耗翻倍，低于四分之一耐久时受到的耐久损耗会越来越少",
   "material.materialis.utherium": "御腐",
   "material.materialis.utherium.encyclopedia": "每级对腐衰生物造成25%的额外伤害",
-  "material.materialis.utherium.format": "御腐%s",
   "modifier.materialis.adrenaline": "肾上腺素",
   "modifier.materialis.adrenaline.description": "生命值越低，攻击伤害越高",
   "modifier.materialis.adrenaline.flavor": "或战或逃！",
   "modifier.materialis.arcane": "奥术",
+  "modifier.materialis.arcane.attack_damage": "魔法伤害",
   "modifier.materialis.arcane.description": "额外对目标造成少量魔法伤害",
   "modifier.materialis.arcane.flavor": "魔法 *哼*",
   "modifier.materialis.auxiliary_power": "备用能源",
   "modifier.materialis.auxiliary_power.description": "采掘速度越高，攻击速度越高",
-  "modifier.materialis.auxiliary_power.flavor": "额外的果汁！",
+  "modifier.materialis.auxiliary_power.flavor": "额外的能源！",
+  "modifier.materialis.bedrock_breaker": "基岩破坏者",
+  "modifier.materialis.bedrock_breaker.description": "使这个工具能破坏基岩",
+  "modifier.materialis.bedrock_breaker.flavor": "打破不可能",
   "modifier.materialis.brittle": "易碎",
   "modifier.materialis.brittle.description": "工具的耐久度会消耗得更快",
   "modifier.materialis.brittle.flavor": "脆得跟粉笔一样",
+  "modifier.materialis.capacitor": "电容",
+  "modifier.materialis.capacitor.description": "工具的能量容量增加 10000RF",
+  "modifier.materialis.capacitor.flavor": "RF 电容，使时间旅行变成可能",
```

```suggestion
  "modifier.materialis.capacitor.flavor": "通量电容，使时间旅行变成可能",
```
叫通量应该也没问题吧（）
毕竟全称就是红石通量